### ISSUE TITLE
warning about uefi 

### ISSUE THREAD

**Issue by carchrae** [devendor/c302ca#1]
firstly; thank you for putting this all down.  it is indeed a nice laptop, however, the sound issue drove me bonkers.  i had it working in 18.04/19.04 but haven't been able to fix it in 20.04.  thank you for the kernel building instructions.  

now, i wanted to convey a warning based on my experience.  i decided to purge chromeos for good, and installed the uefi firmware from https://mrchromebox.tech/#bootmodes (the rabbit/tianocore) and run ubuntu 20.04 that way.  initially, it seemed ok, but the system kept getting disk corrupt errors (crc errors on the journal, ext4), on both the internal drive and the sd card slot.  i tried many, many, (many) things, but nothing would resolve the appearance of write errors.  eventually, i gave up and restored seabios (aka Legacy BIOS) and it has been stable since - and yes, the horrible dev screen, but set to a 1 second pause.

i'm curious if you are running seabios or tianocore.  and if you had any disk errors with tianocore?

**Comment by giacomo-c** [comment]
I've been using Mr. Chromebox's uefi firmware and have been running into problems with disk errors.  e2fsck seems to clean it up, but I didn't realize it was due to the firmware.  I may want to swap back to seabios.

**Comment by carchrae** [comment]
yes, the disk errors seemed to happen intermittently and the fsck fixes are only temporary.  there may be some setting to make it more reliable, but i did not find it.  i tried fstrim options, suspend options, many many things.  i even resorted to using fsck on boot - thinking it might not be a serious error.  it was still very frustrating to use because once an error is detected, the disk goes into read-only mode.

having restored it back to the original legacy (seabios) we have not seen any disk errors since (it has been two weeks)

i hadn't reported it to @MrChromebox as it would be a hard error to reproduce unless you had the same hardware.
